src/easystar.js

```diff
@@ -421,7 +435,11 @@ EasyStar.js = function() {
     };
 
     // Helpers
-    var isTileWalkable = function(collisionGrid, acceptableTiles, x, y) {
+    var isTileWalkable = function(collisionGrid, acceptableTiles, x, y, sourceNode) {
+        if (directionalConditions[x + "_" + y]) {
+            var direction = calculateDirection(sourceNode.x - x, sourceNode.y - y)
+            if (!includes(directionalConditions[x + "_" + y], direction)) return false
```

Comment:
Instead of bringing in a new dependency could we iterate in `for (var i = 0...` style over the directions here ?